README.md

```diff
@@ -262,29 +262,29 @@ Please, have a look at the [`witnet/truffle-box`](https://github.com/witnet/truf
 |  Contract                            ·  Method         ·  Min        ·  Max        ·  Avg        ·  # calls     ·  usd (avg)  │
 ·······································|·················|·············|·············|·············|··············|··············
-|  WitnetProxy                         ·  upgradeTo      ·          -  ·          -  ·     121007  ·           1  ·          -  │
+|  WitnetProxy                         ·  upgradeTo      ·          -  ·          -  ·     121146  ·           1  ·          -  │
 ·······································|·················|·············|·············|·············|··············|··············
-|  WitnetRequestBoardTrustableDefault  ·  deleteQuery    ·      38363  ·      41644  ·      40414  ·           8  ·          -  │
+|  WitnetRequestBoardTrustableDefault  ·  deleteQuery    ·      38352  ·      41633  ·      40403  ·           8  ·          -  │
 ·······································|·················|·············|·············|·············|··············|··············
-|  WitnetRequestBoardTrustableDefault  ·  destroy        ·          -  ·          -  ·      13560  ·           2  ·          -  │
+|  WitnetRequestBoardTrustableDefault  ·  destruct       ·          -  ·          -  ·      13582  ·           2  ·          -  │
 ·······································|·················|·············|·············|·············|··············|··············
-|  WitnetRequestBoardTrustableDefault  ·  initialize     ·          -  ·          -  ·      75004  ·          30  ·          -  │
+|  WitnetRequestBoardTrustableDefault  ·  initialize     ·          -  ·          -  ·      75077  ·          30  ·          -  │
 ·······································|·················|·············|·············|·············|··············|··············
-|  WitnetRequestBoardTrustableDefault  ·  postRequest    ·     144525  ·     196359  ·     164155  ·          33  ·          -  │
+|  WitnetRequestBoardTrustableDefault  ·  postRequest    ·     144650  ·     196484  ·     164280  ·          33  ·          -  │
 ·······································|·················|·············|·············|·············|··············|··············
-|  WitnetRequestBoardTrustableDefault  ·  reportResult   ·     119164  ·     121313  ·     119760  ·          18  ·          -  │
+|  WitnetRequestBoardTrustableDefault  ·  reportResult   ·     119210  ·     121359  ·     119806  ·          18  ·          -  │
 ·······································|·················|·············|·············|·············|··············|··············
-|  WitnetRequestBoardTrustableDefault  ·  upgradeReward  ·      31319  ·      36462  ·      34748  ·           6  ·          -  │
+|  WitnetRequestBoardTrustableDefault  ·  upgradeReward  ·      31341  ·      36484  ·      34770  ·           6  ·          -  │
```

Comment:
I'm kind of missing `deleteQuery` benchmarked here. Do you think it makes sense to add it?